### DISCUSSION TITLE
highlightjs: redundant variable declaration

### DISCUSSION THREAD

**Discussion by oeyoews** [TiddlyWiki/TiddlyWiki5#8046] [General]
<img width="595" alt="image" src="https://github.com/Jermolene/TiddlyWiki5/assets/72405338/f1bfd787-a280-4dcb-9710-00d42709b037">

The hljs here looks repetitive. Was it written this way on purpose? @Jermolene 
https://github.com/Jermolene/TiddlyWiki5/blob/master/plugins/tiddlywiki/highlight/files/tiddlywiki.files

**Comment by oeyoews** [comment]
I took a look [suffix tiddlywiki.files](https://tiddlywiki.com/#:~:text=suffix%20%2D%20(optional)%20a%20string%20to%20be%20appended%20to%20the%20value%20of%20the%20field) In this document, he seems to use eval to re-judge the code, but strangely it always prompts that the exported module cannot be found https://github.com/oeyoews/tiddlywiki-starter-kit/commit/df6759b3c66589e246b395378a65f7736111734d This is really weird strangeness.

**Reply by Jermolene** [reply]
Hi @oeyoews that does look like a mistake. We often have to change the prefix/suffix as imported 3rd party modules change, and I think all that has happened here is that there has been a muddle.

I'd welcome a PR to fix this.
